webpack.config.js

```diff
@@ -59,6 +60,7 @@ const browserConfig = (env, argv) => {
     },
     devtool: env && env.production == 'true' ? 'source-map' : 'eval-source-map',
     module: _module, // module is already defined
+    mode: env && env.production === 'true' ? 'production' : 'development',
```

Comment:
What was the point of changing these?

Reply:
those are editions, it would not build otherwise

Reply:
https://webpack.js.org/configuration/mode/

this will toggle development vs production level optimizations

so for instance in development mode webpack will name chunks and whatnot, vs in production they do module concatenation and other goodies

Reply:
Likely is new between webpack 3 and 4